docs/source/install.rst

```diff
@@ -56,9 +56,27 @@ And then you'll need to install dependencies::
     $ pip install -r dev-requirements.txt
 
 *pythondotorg* will look for a PostgreSQL database named ``pythondotorg`` by
-default. Run the following command to create a new database::
+default. Run the following commands to create a new database.
 
-    $ createdb pythondotorg -E utf-8 -l en_US.UTF-8
```

Comment:
Can we keep this command as default and convert your additional commands to a note?

For example:

```rst
default. Run the following command to create a new database::

    $ createdb pythondotorg -E utf-8 -l en_US.UTF-8

.. note::

   .. You can change the wording, I just use it as an example.

   If you get an error message by running the command above::

       createdb: could not connect to database template1: FATAL:  Peer authentication failed for user "postgres"


   ...

.. Rest of the documentation here.
```